source/vistas/core/gis/elevation.py

```diff
@@ -214,3 +225,31 @@ def measure(lat1, lon1, lat2, lon2):
         task.status = task.INDETERMINATE
         new_plugin.calculate_stats()
         return new_plugin
+
+    def create_data_dem(self, extent, zoom, merge=False):
+        """
+        Creates an elevation grid from elevation data in memory.
+        :param extent Extent to build the DEM for.
+        :param zoom The zoom level to render the DEM at.
+        :param merge Indicate whether to return the DEM as a single numpy grid, or as a cache, using mercantile.Tile
+        objects to index into the cache.
+        :return {mercantile.Tile: numpy.ndarray} if merge==False, numpy.ndarray if merge==True
```

Comment:
OK, I see now that the return type in the comment is more involved.

This can still be handled with type hints:

```python
def create_data_dem(self, extent, zoom, merge=False) -> Union[Dict[mercantile.Tile, numpy.ndarray], numpy.ndarray):
```